Packs/WALLIXBastion/Integrations/WAB/WAB.yml

```diff
@@ -1647,11 +1938,25 @@ script:
     - type: String
       contextPath: WAB.authentication_get.url
       description: The API URL to the resource.
+    arguments:
+    - name: auth_id
+      description: An authentication id. If specified, only this authentication is returned.
+      required: true
+    - name: from_date
+      description: 'Return authentications from this date/time (format: "yyyy-mm-dd" or "yyyy-mm-dd hh:mm:ss").'
```

Comment:
maybe add a validation for the format inside the code? and return an error if not

Reply:
same

Reply:
If the error message is inappropriate, that's fine with me.